programs/mango-v4/src/instructions/token_register_trustless.rs

```diff
@@ -99,10 +105,10 @@ pub fn token_register_trustless(
         collected_fees_native: I80F48::ZERO,
         loan_origination_fee_rate: I80F48::from_num(0.0005),
         loan_fee_rate: I80F48::from_num(0.005),
-        maint_asset_weight: I80F48::from_num(0),
-        init_asset_weight: I80F48::from_num(0),
-        maint_liab_weight: I80F48::from_num(1.4), // 2.5x
-        init_liab_weight: I80F48::from_num(1.8),  // 1.25x
+        maint_asset_weight: I80F48::from_num(0), // 2.5x
+        init_asset_weight: I80F48::from_num(0),  // 1.25x
+        maint_liab_weight: I80F48::from_num(1.4),
+        init_liab_weight: I80F48::from_num(1.8),
```

Comment:
moving these comments is probably an accident?

Reply:
good catch, fixed, I did the vscode struct field reordering, apparently it didnt take the comments with it